developers_chamber/ecs_utils.py

```diff
@@ -559,12 +598,18 @@ def start_service_and_wait_for_tasks_to_start(cluster, service, count, region):
 def get_min_capacity_for_service(cluster, service, region):
     as_client = _get_autoscaling_client(region)
 
-    response = as_client.describe_scalable_targets(
-        ServiceNamespace='ecs',
-        ResourceIds=[
-            'service/{}/{}'.format(cluster, service),
-        ],
-    )
+    try:
+        response = as_client.describe_scalable_targets(
+            ServiceNamespace='ecs',
+            ResourceIds=[
+                'service/{}/{}'.format(cluster, service),
+            ],
+        )
+    except ecs_client.exceptions.ServiceNotFoundException:
```

Comment:
Tady už exception ecs_client.exceptions.ClusterNotFoundException (podobně jako v blocích výše) nastat nemůže?

Reply:
Bohužel nikde není napsané co to může vyhazovat za vyjímky, nicméně když jsem to testoval, tak i když jsem zadal nevalidní cluster, tak to vyhazovalo ServiceNotFoundException. Fallback je ten ClientError.

`'service/{}/{}'.format(cluster, service),` - důvod je, že hledám resource typu service, která je daná i názvem clusteru, nicméně chybou je, že to takovou service nenašlo. Api je to ovšem jiné - ecs_client vs autoscaling_client, takže odpovědi jsou mírně odlišné, tak to funguje v AWS.

Reply:
OK, díky za vysvětlení, já to komentoval pouze na základě toho, že mě zarazila podobnost a jiné výjimky.